### PR TITLE
Fixed classes, added build script

### PR DESCRIPTION
**Pull request by dominikwilkowski**

### CODE REVIEW

File: packages/body/src/sass/module.scss [hunk 1]
```diff
@@ -61,7 +61,7 @@ a {
  */
 mark,
 ::selection {
-	background-color: $uikit-colour-Warning-60;
+	background-color: rgba( $uikit-colour-Warning-60, 0.5 );
```
Comment (klepas) [inline packages/body/src/sass/module.scss]: Mhh?

File: packages/body/tests/site/index.html [hunk 2]
```diff
@@ -117,7 +117,7 @@
 
 	<p>Back to a paragraph.</p>
 
-	<hr />
+	<hr>
```
Comment (klepas) [inline packages/body/tests/site/index.html]: Fair ’nuff — still valid being self-closing (being a void tag) but might as well remain consistent with the general approach html is going in.

File: packages/body/src/sass/module.scss (hunk 1 above)
Comment (dominikwilkowski) [inline packages/body/src/sass/module.scss]: highlights have to be transparent :)
_(or you can't stack them)_

File: packages/body/tests/site/index.html (hunk 2 above)
Comment (dominikwilkowski) [inline packages/body/tests/site/index.html]: Let's not go xhtml... we are more the modern type no? with html5 and all that 🗡

File: packages/body/src/sass/module.scss (hunk 1 above)
Comment (klepas) [inline packages/body/src/sass/module.scss]: Clarifying—this is for when a marked element is selected/highlighted.

Styling can be revisited later.

**Pull request merged**